src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -695,6 +695,10 @@ public boolean isRequired () {
      * FIXME: add foreign reference indexes?
      */
     public void createIndexes(Connection connection, String namespace) throws SQLException {
+        if ("agency".equals(name)) {
+            LOG.info("Skipping indexes for {} table", name);
```

Comment:
This probably doesn't need a template string.

Reply:
I used a template string in case we need to add tables to skip in the future and so that the agency table didn't get hard coded into the log message. Is that a strong enough reason? I realized I could also add a comment here explaining why it's being skipped, which I will do.

Reply:
You mean the template string in the log message? Personally I think it's fine. The overhead from substitution is trivial - the fact of logging at all probably has more impact, and as this is not a hotspot there is no need for optimization. Some people would always use template substitution in log messages for consistency - I often do it.

Reply:
Oh I see, there's only one possible value of `name` because of the conditional. This does make the template seem out of place. Is it expected that there will be more unindexed tables in the future?

Reply:
That was my intention with the use of the template string. I've just added feed_info to the conditional because there is absolutely no reason to index that table (it just wasn't causing problems like the agency index was).